src/db-connection.ts

```diff
@@ -69,6 +69,38 @@ export async function createDbConnection(dbConfig: Db = config.db): Promise<Knex
   return connection
 }
 
+export async function createReplicaDbConnection(
```

Comment:
instead of having a separate function can we reuse the one above , I like the explicit key value pairs

Reply:
Also does this also need to do migrations (in case of future)

Reply:
I like having two separate functions. So that if in the future we want to disable a read replica, it would be easier to add that as a config, and add a check in app.ts. I like having a function do just one thing. 
No we don't need any migrations, rds takes care of that